Platform/Teams/Triage/SupportTiers.md

```diff
@@ -0,0 +1,21 @@
+
+# Tier Definitions
+As part of VSP we will Provide Tier 2/3 support as necessary to resolve platform-originated bugs/issues discovered on the VSP. However, if bugs/issues originate from Veteran-facing services built or maintained by other VA teams Tier 2/3 support will not be required under this contract. 
+
+**Tier 1** Support is staffed by the VA's Contact Centers
+
+**Tier 2** Support is comprised of technicians experienced and knowledgeable about the Veteran-facing Services Platform that can assist end-users to resolve problems. 
+
+**Tier 3** Support is comprised of developers and engineers that analyze technical problems and define root causes. 
```

Comment:
this is specifically the triage team, right?

s/developers and engineers/developers and PMs/ ?

Reply:
This is the exact wording from the PWS, but I don't mind changing it.

Reply:
Oh interesting. Maybe just mention that then?

```
From the PWS, the support tiers are defined as follows:
```